account/guides/setupNetworkBridge.md

```diff
@@ -7,22 +7,30 @@ You must contact [Bandwidth Customer Support](http://support.bandwidth.com/) to
 * [Twilio Account](http://twilio.com/)
 * [Bandwidth Dashboard Account](http://bandwidth.com/)
 * Network Bridge activated: [_contact sales_](https://www.bandwidth.com/)
-* Your customer number IE: `cust11` [_contact support_](http://support.bandwidth.com/)
-  * This will be used as part of the `Realm` combined with `auth.bandwidth.com`
 * Your default port ex: `5006` [_contact support_](http://support.bandwidth.com/)
 
 ## Steps {#steps}
 1. [Create New SIP Credentials](#create-new-sip-credentials)
```

Comment:
In the registrar guide we have step 1 is to create a realm, should that be changed here as well?